bot/resources/utilities/py_topics.yaml

```diff
@@ -109,6 +109,15 @@
     - What books or tutorials would you recommend for game-development beginners?
     - What made you start developing games?
 
+# media-processing
+971142229462777926:
+    - Which Python package/library do you prefer for media processing?
+    - What exactly is media processing and how can Python be used for it?
```

Comment:
I personally think that this question would be better fit for the channel description/a pin.

Reply:
I can see that, let's do a pin. Would you mind if I made it?

Reply:
Something like:

Some common questions related to media processing:
- Which Python package/library do you prefer for media processing?
- What exactly is media processing and how can Python be used for it?

Reply:
> I can see that, let's do a pin. Would you mind if I made it?

I think Zig's message in the pins answers the question about "What exactly is media processing and how can Python be used for it?" already. (that's the one I meant to highlight, but GitHub also showed the other one.

Reply:
Oh, so
* Which Python package/library do you prefer for media processing?
is fine?